MonacoEditorComponent/Themes/generic.xaml

```diff
@@ -15,12 +16,13 @@
                         VerticalAlignment="Stretch">
                         <WebView x:Name="View"
                                  Margin="{TemplateBinding Padding}"
-                                 Visibility="{Binding IsLoaded,RelativeSource={RelativeSource TemplatedParent}}"
+                                 Windows10version1809:Visibility="{Binding IsLoaded,RelativeSource={RelativeSource TemplatedParent}}"
```

Comment:
Why the conditional XAML add here? I hadn't noticed any issues down-level in the past, as I had written this code originally back for 1703.

Reply:
My Visual Studio 2019 threw an warning about it so I applied suggested quick fix.

Reply:
Ah, I see what happened.  I had defined my own `IsLoaded` property for this control specifically before the platform added it to FrameworkElement in 1809, that's why we got the warning about `new` required [here](https://github.com/hawkerm/monaco-editor-uwp/pull/30/files#diff-cf34313236fca092b904331bfffe8a69R33).

We probably want to rename our `IsLoaded` to `IsEditorLoaded`, that's what the Visibility is bound to, so we only show the WebView once the code-editor page is loaded and ready to go, otherwise you get a white flash as the WebView is added to the visual tree but is still loading.

It's always funny when the platform changes underneath you and effects your previously fine code... 😋 At least WinUI 3 will fix that 😊

Reply:
Renamed to `IsEditorLoaded` via 68d6c873

Reply:
Awesome, thanks!